numpy/lib/__init__.py

```diff
@@ -35,13 +35,13 @@
 from . import _npyio_impl
 from . import npyio
 from . import arrayterator
+from . import introspect
 from . import _arraypad_impl
 from . import array_utils
 from . import _version
 
 from .arrayterator import Arrayterator
-from ._version import *
-from .introspect import *
```

Comment:
I just double-checked, there's no need to modify `__getattr__` below since `np.lib.opt_func_info` hasn't made it into a release yet.